README.md

```diff
@@ -39,6 +39,32 @@ Before you get started:
   * `~/.git-credentials` must exist with git credentials. If you are using two-factor auth you will need a create a [personal access token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line)
   * `lint` in $PATH which is provided by having `$GOPATH\bin` in `$PATH` as in `export PATH=$GOPATH/bin:$PATH`.
 
+## Running the Server
+In order to run KUDO, [kubebuilder](https://book.kubebuilder.io/getting_started/installation_and_setup.html) must be installed and in your `PATH` and CRDs must be installed.   CRDs are installed with `make install` or `make manifests` and can be confirmed with `kubectl get crds` resulting in something looking like:
```

Comment:
Why do we need kubebuilder installed after we generate the skeleton?

Reply:
I don't think we need `kubebuilder` installed for this particular section. The make target for `make install` just runs `kubectl` (which should be installed) and `go run` (so obviously Golang should be installed). And the make target for `make run` runs `go generate`, `go fmt` and `go vet` followed by `go run` - none actually require kubebuilder imo.